source/src/BNFC/Backend/TreeSitter/RegToJSReg.hs

```diff
@@ -0,0 +1,94 @@
+{-
+    BNF Converter: TreeSitter Grammar Generator
+    Copyright (C) 2004  Author:  Markus Forsberg, Michael Pellauer,
+                                 Bjorn Bringert
```

Comment:
I guess you want that copyright notice gone...

Reply:
Done